draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -931,6 +933,73 @@ As noted in {{multiplexing}}, all RoQ streams and RoQ datagrams begin with a flo
 
 Because QUIC provides a capability to migrate connections for various reasons, including recovering from a path failure ({{Section 9 of !RFC9000}}), when a QUIC connection migrates, a RoQ sender has the opportunity to revisit decisions about which RTP packets are encapsulated in QUIC streams, and which RTP packets are encapsulated in QUIC DATAGRAMs. Again, RoQ receivers need to be prepared for this eventuality.
 
+## RTCP Considerations
+
+RTCP was originally defined to be used with UDP, which implies (1) that
+transmission timing is largely under the control of the application (limited
+buffering inside the stack) and (2) that the overhead, *avg_rtcp_size*, used to
```

Comment:
```suggestion
RTCP was originally defined to be used with UDP, which implies (1) 
the only buffering present would be at the interface level, so that transmission timing is largely under the control of the application, and (2) that the overhead, *avg_rtcp_size*, used to
```